alerter.c

```diff
@@ -26,6 +33,7 @@ void alertInCelcius(float farenheit) {
 int main() {
     alertInCelcius(400.5);

```

Comment:
At least 3 values need to be checked i.e. if threshold value is 100 C, then input value <100C, 100 C and >100C.
This will clearly state expected behavior of production code.